S21/FaizanSyed95/Language_test.py

```diff
@@ -0,0 +1,64 @@
+import csv 
+
+Vowels=['A','E','I','O','U','Á','É','Í','Ó','Ú']
+GreekCapital=['Π','Ό','Λ','Η','Σ']
+GreekSmall=['π','ό','λ','η','ς']
+
```

Comment:
Did you find a way to implement the other characters? For example, Chinese characters.

Reply:
Chinese, Thai, and Japanese do not have any lower case or upper cases the output for them is the same. So it is easier to just output whatever input you give in it. It is shown below in the function.

Reply:
Okay, that makes sense. I was just wondering why you were calling vowels as a variable.

Reply:
I make that variable name to make it easier for the reviewer to understand that this variable is only be used in the Irish language. That was written in the description of this project.